docs/concepts/automations.md

```diff
@@ -138,14 +134,12 @@ Prefect tries to infer the relevant event whenever possible, but sometimes one d
 
 Specify a name and, optionally, a description for the automation.
 
-![Configuring details for an automation in Prefect Cloud.](/img/ui/automations-details.png)
```

Comment:
Since you're removing this image entirely, which I support, I think you should also remove the "Details" sub-header.